webrtc-stats.html

```diff
@@ -2213,10 +2215,10 @@ <h2>
               </dt>
               <dd>
                 <p class="fingerprint">
-                  Only defined when [= exposing hardware is allowed =].
+                  Omitted unless [= exposing hardware is allowed =].
```

Comment:
Here and other places that use "Omitted": I prefer "Does not [= map/exist =] unless [= exposing hardware is allowed =]." for the same reasons as I preferred "Does not [= map/exist =] for audio."